### PR TITLE
Improve CONTRIBUTING-vi translation

### PR DESCRIPTION
**Pull request by torn4dom4n**
## What does this PR do?
Improve repo

### Description

Using Vietnamese for some words.

### CODE REVIEW

File: CONTRIBUTING-vn.md [hunk 1]
```diff
@@ -2,57 +2,57 @@
 
 Bản dịch Tiếng Việt:
```
Comment (torn4dom4n) [inline CONTRIBUTING-vn.md:3]: Can I remove this? It isn't in official `CONTRIBUTING`.

**Review by torn4dom4n — COMMENTED** [review]
I think `vi` is better than `vn`

**Comment by eshellman** [comment]
whoops! I'm going to revert the change of the page name. 'vi' is the iso language code for vietnamese

**Comment by eshellman** [comment]
whoops again! the change was correct

**Comment by torn4dom4n** [comment]
@eshellman Can I remove "Bản dịch Tiếng Việt". It doesn't show in [CONTRIBUTING](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md). This note isn't necessary.

**Pull request merged**